src/ApiGenerator/RestSpecification/XPack/sql.query.json

```
@@ -1,8 +1,7 @@
 {
   "sql.query":{
     "documentation":{
-      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-rest-overview.html",
-      "description":"Executes an SQL request"
+      "url":"Execute SQL"
```

Bad merge on the ES side?

---

```suggestion
      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-rest-overview.html"
```

The way it is on ES, will use the old url for now